core/backend/src/rpc-impl/IModelTileRpcImpl.ts

```diff
@@ -228,8 +231,8 @@ export class IModelTileRpcImpl extends RpcInterface implements IModelTileRpcInte
 
 /** @internal */
 export async function cancelTileContentRequests(tokenProps: IModelRpcProps, contentIds: TileTreeContentIds[]): Promise<void> {
-  const props: TileContentRequestProps = { accessToken: RpcTrace.currentActivity!.accessToken, tokenProps, treeId: "", contentId: "" };
-  const iModel = await RpcBriefcaseUtility.findOpenIModel(props.accessToken, tokenProps);
+  const iModel = IModelDb.findByKey(tokenProps.key);
+  const props: TileContentRequestProps = { tokenProps, treeId: "", contentId: "" };

```

Comment:
if this is always from Ipc, you can use { accessToken: IModelHost.getAccessToken(), ... } instead of making accessToken possibly undefined.

Reply:
I don't need an access token for Ipc.